ios/RNTextSize.m

```diff
@@ -187,7 +178,9 @@ - (dispatch_queue_t)methodQueue {
   [textStorage addLayoutManager:layoutManager];
 
   NSMutableArray<NSNumber *> *result = [[NSMutableArray alloc] initWithCapacity:texts.count];
-  const CGFloat epsilon = 0.001;
+
+  const CGFloat scaleMultiplier = _bridge ? _bridge.accessibilityManager.multiplier : 1.0;
+  const CGFloat epsilon = scaleMultiplier != 1.0 ? 0.001 : 0;
```

Comment:
Can you add a comment about how you figured out the epsilon on the lineHeight?

Reply:
Admittedly, this was found empirically when playing around with enabling/disabling epsilon under certain conditions and found that this happened to work. Added the comment